dags/example_dags/example_bash.py

```diff
@@ -0,0 +1,9 @@
+# from airflow.models import DAG
```

Comment:
This is the weird issue i'm fighting.  If i remove this line ... the dag doesn't show up in the Airflow UI